packages/wrangler/src/__tests__/index.test.ts

```diff
@@ -1,9 +1,9 @@
 import * as fs from "node:fs";
 import * as fsp from "node:fs/promises";
-import * as TOML from "@iarna/toml";
 import { parseConfigFileTextToJson } from "typescript";
 import { version as wranglerVersion } from "../../package.json";
 import { getPackageManager } from "../package-manager";
+import { parseTOML } from "../parse";
```

Comment:
I don't think we should use this function in tests. We silence logs in tests so there's no benefit to a better message, there's no user facing bits here.

Reply:
I figured we could mock out or blackhole the errors in tests. I can revert the test files.